### PR TITLE
removes the word daddy and mommy

### PR DESCRIPTION
**Pull request by Bibby0110**
# Document the changes in your pull request

# Wiki Documentation

# Changelog

:cl:  
rscadd: desexualizes the game
/:cl:

### CODE REVIEW

**Comment by num1bamf** [comment]
Yes daddy

**Comment by SuperTeaLove** [comment]
please stop aaaaaaa

good pr though

**Comment by ToasterBiome** [comment]
What about mommy?

File: code/game/say.dm [hunk 1]
```diff
@@ -26,6 +26,8 @@ GLOBAL_LIST_INIT(freqtospan, list(
 		return
 	if(message == "" || !message)
 		return
+	if(message == "daddy")
```
Comment (ToasterBiome) [inline code/game/say.dm]:
```suggestion
	if(message == "daddy" || message == "mommy")
```

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

**Comment by ToasterBiome** [comment]
Ok no more funny

**Comment by N3D6** [comment]
Unironically good pr

**Pull request closed without merging**